packages/storybook/.storybook/manager.js

```diff
@@ -1,4 +1,4 @@
-import { addons } from '@storybook/manager-api';
```

Comment:
Storybook 9 makes a [bunch of these package changes](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropped-support-for-legacy-packages)